attestation-manager/src/cmd/app/main.go

```diff
@@ -218,7 +218,29 @@ func retryOperation(operation func() (bool, interface{}), retryCount int, retryI
 	return false, nil
 }
 
-func cordonDrainNode() {
+func handleErrorDetails(cfg *constants.Config, attestToken string) {
+	logging.Info("Handling error: logging flavor templates and trust report")
+
+	logging.Info("Fetching flavors...")
+	flavors, err := api.GetFlavorIDs(cfg, attestToken)
+	if err != nil {
+		logging.Error("Error fetching flavors:", err)
+	} else {
+		logging.Info("Flavors:", flavors)
+	}
+
+	logging.Info("Fetching trust report...")
+	success, trustReport := api.FetchTrustReport(cfg, attestToken, cfg.TCHOSTNAME)
+	if !success {
+		logging.Error("Error fetching trust report")
+	} else {
+		logging.Info("Trust Report:", trustReport)
```

Comment:
is there any confidential info/data we need to be concern with in the trustReport that might need to be filter out (e.g credentials, passwords, etc)

Reply:
There is no confidential data associate with reports.